includes/class-mf2-debugger.php

```diff
@@ -0,0 +1,120 @@
+<?php
```

Comment:
As far as I can see, it is verified, except for the MF2 Parser, which isn't under my control.